build.gradle

```diff
@@ -78,6 +82,7 @@ dependencies {
   testImplementation("org.springframework.boot:spring-boot-starter-test") {
     exclude group: "org.junit.vintage", module: "junit-vintage-engine"
   }
+  testImplementation("com.willowtreeapps.assertk:assertk-jvm:0.21")
```

Comment:
What is willowtreeapps? Just the creator of assertk? I've never heard of it

Reply:
It's just the creator of assertk (which is just a kotlin wrapper around assertj)